stories/topics/tech-note-aws-backup-restore.adoc

```diff
@@ -0,0 +1,5 @@
+[id="tech-note-aws-backup-restore"]
+
+= Backup and Restore
+
+The aws_rds_db_snapshot_arn optional parameter which allowed customers to specify a specific AWS RDS Snapshot to use for restoring their deployment is no longer supported. The restore process no longer uses AWS RDS Snapshots to restore a deployment and instaed relies on the backup files generated by the Ansible on Clouds Ops Container backup playbook. The Ansible on Clouds Ops Container restore playbook will not use any provided RDS snapshot passed as a parameter. 
```

Comment:
aws_rds_snapshot_arn enclose in backticks

and instead relies

restore playbook does not use any